README.rst

```diff
@@ -150,8 +135,8 @@ Create a conda environent (using Python 3 here) and install it in development mo
 
     $ conda create -n esss-fix-format-py3 python=3
     $ source activate esss-fix-format-py3
-    $ pip install -e .
     $ pip install . -r requirements_dev.txt
+    $ pip setup.py develop
```

Comment:
Typo: `python setup.py develop`.

But actually `pip install -e .` is recommended over `python setup.py develop`.

I suggest to install this with `requirements_dev.txt` then:

```
pip install -e . -r requirements_dev.txt
```

Reply:
Changed.